lib/ansible/modules/storage/glusterfs/gluster_peer.py

```diff
@@ -145,16 +146,36 @@ def _run_command(self, op, opts):
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            force=dict(choices=["yes", "no"]),
+            force=dict(type='bool', required=False),
             nodes=dict(),
             state=dict(required=True, choices=["absent", "present"]),
         ),
+        supports_check_mode=False,
     )
     pops = Peer(module)
+    # Verify if GlusterFS 3.2 or over is installed
+    if not valid_gluster_version(module):
+        module.fail_json(msg="GlusterFS version > 3.2 is required")
     cmds = pops.gluster_peer_ops()
     errors = pops.call_peer_commands(cmds)
     pops.get_output(errors)
 
 
+def valid_gluster_version(module):
+    cmd = module.get_bin_path('gluster', True) + ' --version'
+    # Check if the required gluster version is installed
+    result = module.run_command(cmd)
+    ver_line = result[1].split('\n')[0]
+    version = ver_line.split(' ')[1]
+    major_vers = int(version[0])
+    minor_vers = int(version[2])
```

Comment:
it should be `minor_vers = int(version[1])`

Reply:
I'll use LooseVersion